src/Pachyderm/Orm/Model.php

```diff
@@ -348,11 +348,18 @@ public static function pagination(array $params): SQLBuilder
         unset($params['order']);
 
         if (!empty($params['filter'])) {
-            $query = new QueryBuilder($params['filter']);
+            $conditions = json_decode($params['filter']);
+            $query = new QueryBuilder();
+            foreach ($conditions as $condition) {
```

Comment:
This makes a different behavior of what was done before.

Reply:
The previous pass filter seems doesn't work and not used anywhere else. The filter here is a special reserved key for those kind of queries.

Reply:
This doesn't work as it was before. You've destroyed a working feature on a framework used by many apps.